proto/src/main/proto/Rss.proto

```diff
@@ -242,6 +249,7 @@ message ShuffleServerHeartBeatRequest {
   int32 eventNumInFlush = 5;
   repeated string tags = 6;
   google.protobuf.BoolValue isHealthy = 7;
+  ServerStatus status = 8;
```

Comment:
Is this a compatible feature? Can the new coordinator receive old shuffle server's heartbeat request?

Reply:
Updated